source/_views/landingpage.html

```diff
@@ -173,7 +173,7 @@ <h3 class="h4">{{ site.text.uoso.feature_1.title|raw }}</h3>
   </div>
 </section>
 
-
+<!--
```

Comment:
is this commented out on purpose? imho we should delete it or use something like twig so its not output to the client at all, if its not meant to be public

Reply:
deleted it